Functions.Templates/Templates/DurableFunctionsActivity-Python-1.x/metadata.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "defaultFunctionName": "Hello",
+  "description": "$DurableFunctionsActivity_description",
+  "name": "Durable Functions activity (upgrade to bundles V2)",
+  "language": "Python",
+  "category": [
+    "$temp_category_core",
+    "$temp_category_durableFunctions"
+  ],
+  "categoryStyle": "other",
+  "enabledInTryMode": false,
+  "userPrompt": [],
+  "extensions": [
+    {
+      "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask", "version": "2.4.0"
+    }
```

Comment:
Hmm, this bit might be wrong.....Can't remember how to set this right. I'll investigate